### PR TITLE
[fr] follows 3.3 migration

### PR DESCRIPTION
**Pull request by cake17**
some parts still need to be translated

### CODE REVIEW

File: fr/appendices/3-3-migration-guide.rst [hunk 1]
```diff
@@ -4,19 +4,68 @@
...
   paramètre réduit le recours à l'état global, et est nécessaire pour le travail
   d'intégration de la norme PSR7.
+- Quand vous construisez vos resource routes, vous pouvez maintenant définir un
+  préfixe. C'est utile quand vous définissez des ressources imbriquées car vous
+  pouvez créer des controllers spécialisés pour les ressources imbriquées.
+
+Console
+=======
+
+- Les Shell tasks qui sont appelés directement à partir du CLI n'appellent plus
```
Comment (antograssiot) [inline fr/appendices/3-3-migration-guide.rst]: appelées

File: fr/appendices/3-3-migration-guide.rst [hunk 2]
```diff
@@ -4,19 +4,68 @@
...
+  la méthode ``_welcome``. Ils vont maintenant aussi avoir le paramètre
+  ``requested`` défini.
+
+Request
+=======
+
+- ``Request::is()`` et ``Request::addDetector()`` supportent maintenant des
+  arguments supplémentaires dans les détecteurs. Cela permet aux detector
```
Comment (antograssiot) [inline fr/appendices/3-3-migration-guide.rst]: detector -> détecteurs

File: fr/appendices/3-3-migration-guide.rst [hunk 3]
```diff
@@ -4,19 +4,68 @@
...
+==========
+
+- ``Validator::requirePresence()`` accepte maintenant une liste de champs. Cela
+  vous permet de définir de façon plus concise les champs qui sont nécessaires.
+
+Debugging Functions
+===================
+
+- Les fonctions ``pr()``, ``debug()`` et ``pj()`` retournent maintenant la
+  valeur étant dumped. Cela facilite leur utilisation quand des valeurs sont
```
Comment (antograssiot) [inline fr/appendices/3-3-migration-guide.rst]: "la valeur. " ou quelquechose comme "la valeur resultante"

**Comment by antograssiot** [comment]
@cake17 looks good so far except my comments. Sorry I don't have much time to help you with tranlation at the moment.

**Comment by cake17** [comment]
@antograssiot thanks for the review :smile: , don't worry there is not so much translation at the moment

**Pull request merged**